test/should_pass/glb_ann_types.erl

```diff
@@ -0,0 +1,20 @@
+-module(glb_ann_types).
+
+-compile(export_all).
+
+-type type() :: erl_parse:abstract_type().
```

Comment:
`?TYPE_MOD` here?

Reply:
indeed. (but this whole testcase is just temporary to help track down the issue)